rpcs3/Emu/Cell/PPUFunction.cpp

```diff
@@ -27,6 +37,7 @@ extern std::string ppu_get_syscall_name(u64 code)
 	case 30: return "_sys_process_get_paramsfo";
 	case 31: return "sys_process_get_ppu_guid";
 	case 41: return "_sys_ppu_thread_exit";
+	case 42: return "sys_ppu_thread_get_id";
```

Comment:
Someone got confused I think, theres no such syscall because its an liblv2.sprx function and it uses no syscalls.